lib/matplotlib/axes/_axes.py

```diff
@@ -3811,8 +3811,7 @@ def scatter(self, x, y, s=None, c=None, marker=None, cmap=None, norm=None,
                 vmin=None, vmax=None, alpha=None, linewidths=None,
                 verts=None, edgecolors=None,
                 **kwargs):
-        """
-        Make a scatter plot of x vs y, where x and y are sequence like objects
+        """Make a scatter plot of x vs y, where x and y are sequence like objects
```

Comment:
I'm pretty sure this will break something, though it may be the `dedent_interpd` that was removed...